### ISSUE TITLE
Limited number of Steps

### ISSUE THREAD

**Issue by hang-air** [rozPierog/Cofi#122]
Hi

I like the cofi app :-)

I wanted to add the new «A better one cup V60» receipe from James Hoffman.

I can only add 7 steps. Afterwards, when I add another step, the edit step is added, thejedit penel is quickly shown but collapses immediately and the stepmis removed again.

Using version 1.11.1 from Fdroid

**Comment by hang-air** [comment]
Oh... Looks like it has to do with the Keyboard. The issue only occurs when I use the TypeWise keyboard. With the gBoard it works fine.

**Comment by rozPierog** [comment]
That's an interesting issue @hang-air . Could I ask you to record a video of what's happening so I can investigate? 
In a meantime [here is a backup json](https://gist.githubusercontent.com/rozPierog/fe11ef36d80e79574dbb712cb5320ea3/raw/947904bced077315ab54a8246aa105b21a6f8b7e/cofi_single_v60.json) of new Hoffmann V60, you can import it into your Cofi by going to Settings>Backup/Restore>Restore
You can download it by `save link as`

**Comment by hang-air** [comment]
Thanks for the backup file, that worked.

Here's a screen capture:

https://user-images.githubusercontent.com/8083550/205337112-55b6e62d-0e39-4420-b7fc-2f6bc7af7f9b.mp4


Thanks!!

**Comment by sandorex** [comment]
I have the same issue but im using Unexpected Keyboard

After a certain number of steps the keyboard won't popup anymore and i cant add any more steps

**Comment by rozPierog** [comment]
@hang-air @sandorex I've submitted a beta to the play store with a potential fix. Google says that it's live but it doesn't show up on my devices. If you could check if this version is working for you that would help me a ton.

**Comment by zzrowla** [comment]
Seems to be working fine over here after the beta update!

**Comment by hang-air** [comment]
Works fine here as well, although I'm not really sure if I really have the right version. It still displays 1.11.1 as before...

Works with GBoard and Typewise

Thank you very much for fixing this!

**Comment by rozPierog** [comment]
Yeah I forgot to update the version number for this beta release 😅 sorry for the confusion. 
Thanks for checking it out. I'll package it into proper release and ship to public release so that it's fixed for everyone.
